tests/test_shieldhit_config_mappings.py

```diff
@@ -96,3 +100,20 @@ def test_energy_cutoff() -> None:
 
     beam.energy_high_cutoff = None
     assert "TCUT0" not in str(beam)
+
+@pytest.mark.parametrize('value_x, value_y',[
+    (None, None),
+    (2, None),
+    (1, 2),
+])
```

Comment:
Make tests with more meaningful parameters, like 200 and 250.

Reply:
Done.